src/tailscale/cli.ts

```diff
@@ -379,6 +419,13 @@ export class Tailscale {
     });
   }
 
+  async snooze() {
+    this.snoozing = true;
+    setTimeout(() => {
```

Comment:
This is only going to snooze within the instance of VS Code it was triggered in. I would like a prompt for the duration, including day, week, and month. For this, I am thinking we could use `context.globalStoragePath` and persist the port config (snooze info) to disk.

Thoughts @samlinville @christine-tailscale

Reply:
When does this notification show up? Will the snooze function delay just this message or will it snooze all notifications from us?

I'm assuming that users would want to snooze because they're in the middle of something else and want to save it for later. What if instead of offering them custom snooze options, we offer them a quick yes/no decision? And then if they click no, we can use that notification to educate them where port discovery lives if they want to come back to it later?

Reply:
The notification shows up as part of port discovery and as this PR stands snoozes discovery entirely:

<img width="559" alt="image" src="https://github.com/tailscale-dev/vscode-tailscale/assets/40265/2ecef5d8-1839-4a88-b55b-e75a02ccac37">

Reply:
We should persist in the requested snooze duration (storing the time when we can _unsnooze_). We should not offer a calendar or complex form to fill out (we're trying to work around an annoyance, let's not annoy them more). Let's keep it simple.

15 minutes seems too short, TBH. The goal is to provide a reasonable snooze option that keeps the port disco from annoying users. If snooze is too short, we're still annoying them. The minimum risk is that the person will disable port disco notifications altogether. The max is the uninstall.

I wouldn't oppose a single "Snooze for 1hr" or "Snooze" and then prompting with 1 hour, 2 hours, or this session. (For example)